modules/nw-anp-banp-metrics.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included:
+// networking/network_security/AdminNetworkPolicy/nw-ovn-k-anp-banp-metric.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="anp-banp-metrics_{context}"]
+= Metrics
```

Comment:
Perhaps this could be a bit more descriptive -  maybe "AdminNetworkPolicy metrics" or something ?